src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -236,7 +236,7 @@ public void datasetTest() throws Exception {
             fromDB.getAssetSpecifications().forEach(this::assertAssetSpecs);
 
             for (GoogleCloudResource resource: GoogleCloudResource.values()) {
-                GoogleRegion region = fromDB.getDatasetSummary().getStorageResourceRegion(resource);
+                GoogleRegion region = (GoogleRegion) fromDB.getDatasetSummary().getStorageResourceRegion(resource);
```

Comment:
I think these cases can be converted to use `CloudRegion`, so the cast can be avoided.